src/binaryutils.cpp

```diff
@@ -225,7 +225,7 @@ InferenceEngine::Blob::Ptr createBlobFromMats(const std::vector<cv::Mat>& images
     InferenceEngine::TensorDesc desc{tensorInfo->getPrecision(), dims, InferenceEngine::Layout::ANY};
     InferenceEngine::Blob::Ptr blob = InferenceEngine::make_shared_blob<T>(desc);
     blob->allocate();
-    char* ptr = (char*)blob->buffer();
+    char* ptr = InferenceEngine::as<InferenceEngine::MemoryBlob>(blob)->rmap().as<char*>();
```

Comment:
Instead of casting Blob to MemoryBlob which inherits from Blob maybe it would be better idea to start using MemoryBlob? What is the point now in using Blob if all the calls require casting to MemoryBlob first. @atobiszei @mkuczyns11

Reply:
I am not familiar with the scope of such a change, so maybe it would be better if @atobiszei says whether that should be changed.